test/journeys/specs/oneOnOne/dataApi/startup-settings.js

```diff
@@ -0,0 +1,110 @@
+import testUsers from '@ciscospark/test-helper-test-users';
+
+import {elements} from '../../../lib/test-helpers/space-widget/main.js';
+import {answer, hangup} from '../../../lib/test-helpers/space-widget/meet.js';
+
+describe(`Widget Space: One on One`, () => {
+  describe(`Data API Settings`, () => {
+    const browserLocal = browser.select(`browserLocal`);
+    const browserRemote = browser.select(`browserRemote`);
+    let mccoy, spock;
+
+    process.env.CISCOSPARK_SCOPE = [
+      `webexsquare:get_conversation`,
+      `Identity:SCIM`,
+      `spark:people_read`,
+      `spark:rooms_read`,
+      `spark:rooms_write`,
+      `spark:memberships_read`,
+      `spark:memberships_write`,
+      `spark:messages_read`,
+      `spark:messages_write`,
+      `spark:teams_read`,
+      `spark:teams_write`,
+      `spark:team_memberships_read`,
+      `spark:team_memberships_write`,
+      `spark:kms`
+    ].join(` `);
+
+    before(`load browsers`, () => {
+      browser
+        .url(`/data-api/space.html`)
+        .execute(() => {
+          localStorage.clear();
+        });
+    });
+
+
+    before(`create spock`, () => testUsers.create({count: 1, config: {displayName: `Mr Spock`}})
+    .then((users) => {
+      [spock] = users;
+    }));
+
+    before(`create mccoy`, () => testUsers.create({count: 1, config: {displayName: `Bones Mccoy`}})
+    .then((users) => {
+      [mccoy] = users;
+    }));
+
+    before(`pause to let test users establish`, () => browser.pause(5000));
+
+    describe(`initial activity setting`, () => {
+      before(`inject token`, () => {
```

Comment:
Maybe create two `describe` blocks for the two different initial activity settings, `meet` or `message`.